File: Iniciante/1015/1015.c
#include <stdio.h>
#include <math.h>

int main() 
{
   float x1, y1, x2, y2, p1, p2, dist;

   scanf("%f %f", &x1, &y1);

   scanf("%f %f", &x2, &y2);

   p1 = x2-x1;

   p2 = y2-y1;

   dist = sqrt((p1*p1) + (p2*p2));
   
   //dist = sqrt(pow(x2-x1,2)+pow(y2-y1,2));

   printf("%.4f\n", dist);

   return (0);
}

File: Python/Iniciante/1015.py
# Problema 1015 - URI - Iniciante - Nível 1

# Adicionando "math" para usar "sqrt"
from math import sqrt

# Lendo x1 e y1
x1, y1 = map(float, input().split(" "))

# Lendo x2 e y2
x2, y2 = map(float, input().split(" "))

# Imprimindo a distância
print(f'{(sqrt((x2 - x1)**2 + (y2 - y1)**2)):.4f}')
